backend/app/services/adapters/task_kinds/operations.py

```
@@ -0,0 +1,947 @@
+        create_subtasks(db, task, team, user.id, obj_in.prompt)
+
+        db.commit()
+        db.refresh(task)
+
+        return convert_to_task_dict(task, db, user.id)
+
+    def _handle_existing_task(
+        self,
+        db: Session,
+        existing_task: TaskResource,
+        obj_in: TaskCreate,
```

Ruff (0.14.13)

Unused method argument: `obj_in`

(ARG002)